cwltool/argparser.py

```diff
@@ -37,6 +37,13 @@ def arg_parser() -> argparse.ArgumentParser:
         help="Output directory. The default is the current directory.",
     )
 
+    parser.add_argument(
+        "--log-dir",
+        type=str,
+        default="",
+        help="Log your tools stdout/stderr to this location outside of container",
```

Comment:
Is `--log-dir` supposed to work for tools that output to stdout/stderr but we don't capture that in the `CommandLineTool` description itself? This help text implies that it does, but my testing says it does not currently do so.

Reply:
I think you need either a stdout/stderr field in the CommandLineTool description or the logs are captured as an output.